src/components/SongPlayer.js

```diff
@@ -91,12 +95,36 @@ function SongPlayer() {
     // logic to play for next song
     React.useEffect( () => {
         const nextSong = data.playlist[postionInPlaylist + 1]
+
         // After finishing, played song becomes 1
-        if( played >= 0.99 && nextSong) {
-            setPlayed(0)
-            dispatch( {type: 'SET_SONG', payload: { song: nextSong }})
+
+        /* WHOLE PLAYLIST REPEAT */
+        // if( played >= 0.99 && nextSong) {
+        //     setPlayed(0)
+        //     dispatch( {type: 'SET_SONG', payload: { song: nextSong }})
+        // }
+        // else if( played >= 0.99 && state.isRepeating ) {
+        //     setPlayed(0)
+        //     if(!firstSong) {
+        //         reactPlayerRef.current.seekTo(played);
+        //     }
+        //     else {
+        //         dispatch( {type: 'SET_SONG', payload: { song: firstSong }})
+        //     }
+        // }
```

Comment:
@zee-bit I mean this part, it's better to remove, I guess

Reply:
Ohh...Yes..yes. I'll remove that part. LOL